modules/backend/classes/BackendController.php

```diff
@@ -205,8 +206,26 @@ protected function getRequestedController($url)
         /*
          * Look for a Plugin controller
          */
-        if (count($params) >= 2) {
-            list($author, $plugin) = $params;
+        if (count($params) >= 1) {
+            $pluginVersions = PluginVersion::applyEnabled()
```

Comment:
@DanHarrin Is there any reason for using `PluginVersion` over just looping through the results of `PluginManager::getPlugins()`? That should also contain the enabled plugin codes, and would save you an SQL query.

Reply:
just pushed changes using that method instead @bennothommo, URLs aren't rewriting now though